packages/reference/src/__fixtures__/space/index.ts

```diff
@@ -0,0 +1,3 @@
+import indifferent from './indifferent_space.json';
+
+export { indifferent };
```

Comment:
Not an issue, but I wonder, why call it "indifferent"?

Reply:
It's indifferent from the current space 🤷🏻‍♂️